### PR TITLE
Fix some formatting

### PR DESCRIPTION
**Pull request by nhakmiller**
### Background

Fix some formatting issues.

### Changes

* Ran `make fmt test`
* Updated make file to apply formatting & linting to  templates

### Testing

* `make test`

### CODE REVIEW

File: templates/example_rule.py [hunk 1]
```diff
@@ -1,37 +1,61 @@
...
 
+
 ## Optional Functions
-def title(event): # User can set custom alert titles in string format
-    return 'This is my title'
+# Set custom alert titles, must return a string
+# If not defined, defaults to the rule display name or ID
+def title(event):
+    return "This is my title " + event.get("Something")
+
```
Comment (wey-chiang) [inline templates/example_rule.py:17]: nit: f-strings
```suggestion
def title(event):
    return f"This is my title {event.get('Something')}"

```

File: templates/example_rule.py [hunk 2]
```diff
@@ -1,37 +1,61 @@
...
 
-def dedup(event): # by default if not defined the deduplication period string is based off the alert title 
-    return 'DedupString'
+# Set custom deduplication strings, must return a string
+# If not defined, defaults to the alert title
+def dedup(event):
+    return "DedupString:" + event.get("Something")
```
Comment (wey-chiang) [inline templates/example_rule.py:22]: nit: f-strings
```suggestion
def dedup(event):
    return f"DedupString: {event.get('Something')}"
```

File: templates/example_rule.py [hunk 3]
```diff
@@ -1,37 +1,61 @@
...
+    return "https://some.com/reference/" + event.get("Something")
 
-def description(event): # customize the description to the specific event (str)
-    return 'Some Alert Description'
 
-def reference(event): # customize the reference to the specific event (str)
-    return 'https://some.com/reference/url'
+# Override the runbook in the alert, must return a string
+def runbook(event):
+    return "If this happens, do " + event.get("Something")
```
Comment (wey-chiang) [inline templates/example_rule.py:51]: nit: f-strings
```suggestion
def runbook(event):
    return f"If this happens, do {event.get('Something')}"
```

File: templates/example_rule.py [hunk 4]
```diff
@@ -1,37 +1,61 @@
...
+
+
+# Override the description of the alert, must return a string
+def description(event):
+    return "Some Alert Description " + event.get("Something")
+
+
+# Override the reference in the alert, must return a string
+def reference(event):
+    return "https://some.com/reference/" + event.get("Something")
```
Comment (wey-chiang) [inline templates/example_rule.py:46]: nit: f-strings

```suggestion
def reference(event):
    return f"https://some.com/reference/{event.get('Something')}"
```

File: templates/example_rule.py [hunk 5]
```diff
@@ -1,37 +1,61 @@
...
+# must return one of the following strings "INFO", "LOW", "MEDIUM', "HIGH", "CRITICAL"
+def severity(event):
+    if event.get("Something"):
+        return "INFO"
+    return "HIGH"
+
+
+# Override the description of the alert, must return a string
+def description(event):
+    return "Some Alert Description " + event.get("Something")
```
Comment (wey-chiang) [inline templates/example_rule.py:41]: nit: f-strings

```suggestion
def description(event):
    return f"Some Alert Description {event.get('Something')}"
```

**Review by wey-chiang — APPROVED** [review]
Some nits for promoting the usage of f-strings but LGTM

File: Makefile [hunk 6]
```diff
@@ -7,7 +7,7 @@ rev := $(shell git rev-parse HEAD)
 # Release tag for current commit
 release := $(shell git tag --points-at=$(rev) --sort=version:refname --list 'v*' | tail -n1)
 
-dirs := $(shell ls | egrep 'policies|rules|helpers|models' | xargs)
+dirs := $(shell ls | egrep 'policies|rules|helpers|models|templates' | xargs)
```
Comment (lindsey-w) [inline Makefile:10]: nice catch

**Review by lindsey-w — APPROVED** [review]

**Pull request merged**